sdk/messaging/azservicebus/example_receiver_test.go

```diff
@@ -86,3 +86,29 @@ func ExampleReceiver_ReceiveMessages() {
 		fmt.Printf("Received and completed the message\n")
 	}
 }
+
+func ExampleReceiver_DeadLetterMessage() {
+	// DeadLetterMessage settles a message by moving it to the dead letter queue.
```

Comment:
What do you think about adding a link to the the example that shows how to create the dead letter receiver?

https://github.com/Azure/azure-sdk-for-go/blob/e0083cee1527d50739c996af139d4ab6065e848c/sdk/messaging/azservicebus/example_receiver_test.go#L36
(not with that exact link, but one to main)

Reply:
I think that's a great suggestion. Creating a receiver is the natural next question the reader would ask. Link added. :)